src/js/components/Video/stories/typescript/ControlsBelow.tsx

```diff
@@ -25,7 +25,13 @@ const SimpleVideo = props => (
   </Grommet>
 );
 
-export const ControlsBelow = () => <SimpleVideo controls="below" />;
+export const ControlsBelow = () => (
+  <SimpleVideo
+    controls={{
+      position: 'below',
+    }}
+  />
```

Comment:
```suggestion
  <SimpleVideo controls='below' />
```
We should be able to leave this as controls='below' for backwards compatibility. Right now if we do that it shows the controls 'over'.